INSTALL.md

```diff
@@ -237,7 +237,7 @@ dnf module install nodejs:12
 #### Dependencies
 
 ```shell
-sudo yum install patch autoconf automake bison libffi-devel libtool libyaml-devel readline-devel sqlite-devel zlib-devel openssl-devel ImageMagick ImageMagick-devel mysql-devel postgresql-devel chromedriver memcached
+sudo yum install patch autoconf automake bison libffi-devel libtool libyaml-devel readline-devel sqlite-devel zlib-devel openssl-devel gd-devel mysql-devel postgresql-devel chromedriver memcached
```

Comment:
We should update macOS setup steps. I can help you test it.

Reply:
Good point!

I see two lines: 
```
brew install imagemagick@6 mysql@5.7 gs pkg-config openssl geckodriver postgresql memcached
brew link imagemagick@6 --force
```

I think we need to replace `imagemagick@6` with `gd`. But doe we need `brew-link`? Could you try this out?

Reply:
I updated instructions for brew. Test, pretty please :pleading_face: